community/case-studies/wayfair/000_index.md

```diff
@@ -0,0 +1,184 @@
+# Wayfair
+
+![wayfair](./media/wayfair.png)
+
+Development on tremor started in the autumn of 2018 as a point solution
+for introducing a configurable traffic shaping mechanism for our
+infrastructure platform engineering organization.
+
+There were two use cases being considered at the time - this use case
```

Comment:
I find this phrasing a little confusing. How about something like, "There were two use cases being considered at the time -  the traffic-shaping use case, which had the highest priority, and the distributed data replication use case for our Kafka-based environments."

Reply:
Agreed, added your rephasing!